curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-basic-algorithmic-thinking-by-building-a-number-sorter/6406bb32f9ed593f26c33b2b.md

```diff
@@ -0,0 +1,309 @@
+---
+id: 6406bb32f9ed593f26c33b2b
+title: Step 6
+challengeType: 0
+dashedName: step-6
+---
+
+# --description--
+
+Remember that `.getElementsByClassName()` returns an array-like object. You can use the spread operator to convert it into an array.
```

Comment:
Here, you are referencing the method itself. I don't think you need to prefix the method with `.`. So, I think it should be `getElementsByClassName()` instead of `.getElementsByClassName()`.

Reply:
I've generally taken the style of prefixing methods with `.` so that there is clarity between what is a method and what is a function.

Reply:
I see